clients/src/test/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumerTest.java

```diff
@@ -1676,7 +1666,8 @@ public void testEnsurePollEventSentOnConsumerPoll() {
                 mock(ConsumerRebalanceListenerInvoker.class),
                 subscriptions,
                 "group-id",
-                "client-id");
+                "client-id",
+            false);
```

Comment:
is indentation off here? (actually I think the "false" is the one that's ok, with 4 spaces, vs all the others with 8. I think we usually do 4 here)